nix/shell.nix

```diff
@@ -0,0 +1,108 @@
+{
+  pkgs,
+  flake
+}:
+
+let
+  qtCustom = (with pkgs.qt515_8;
```

Comment:
Another point in case, the latest released Qt version is 5.15.15, why do we have to use sth that's years old?

Reply:
This was originally requested by:
https://github.com/status-im/status-desktop/issues/9350

Switching to newer version will not be a problem. It's a next step.

Reply:
If with switching to Qt 5.15.15 I will not need to implement and opt-out logic (to build and run with system libs) - I would certainly prefer this.

Reply:
Deal ;) I can test this locally on my machine next week, and provide patches/PR on top of this PR if needed

Reply:
haha, nice middle ground! 😆 Can't wait for the next iteration, on windows this time.😄